src/common/event.cpp

```diff
@@ -1882,24 +1882,29 @@ bool wxEvtHandler::SearchDynamicEventTable( wxEvent& event )
     wxCHECK_MSG( m_dynamicEvents, false,
                  wxT("caller should check that we have dynamic events") );
 
+    // extend the life of the wxRecursionGuardFlag
+    wxSharedPtr<DynamicEvents> addRef(m_dynamicEvents);
```

Comment:
I'd call this `localCopy` or something like this.

Reply:
Fixed in https://github.com/wxWidgets/wxWidgets/pull/25336/commits/b318c500f7c33c7b2c49a056f1a883612c18b9ee